kivy_ios/toolchain.py

```diff
@@ -1365,7 +1365,8 @@ def build(self):
         if ctx.use_pbzip2:
             logger.info("Using pbzip2 to decompress bzip2 data")
 
-        build_recipes(args.recipe, ctx)
+        recipe = ' '.join(args.recipe).replace(',', '').replace('  ', ' ').split()
```

Comment:
Easy and effective, but looks quite hacky and I don't feel super-confident about merging it. (And should be handled on the argparser side instead)

How about an argument `-r` or `--requirement` which parses a `requirements.txt` file as `pip` does ?

Reply:
Sounds good, why not, I'll try to implement